.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Node CI
+
+on:
```

Comment:
What's the reasoning to restrict this only on PRs and commits to master? Why not only `on: [push]`? This way we can test CI on those weird bugs that only happen on CI without having to open a PR.

Reply:
You're right. We can get rid of this constraint. Lack of knowledge from my side on setting up CI triggers

Reply:
All done! ✅